.history/force-app/main/default/classes/NewOfferConsistencyCheckTest_20210713113245.cls

```diff
@@ -0,0 +1,102 @@
+/** 
+ * @author Mikołaj Lechtański <mikolaj.lechtanski@accenture.com> 
+ * @date 07.07.2021 
+ * @description Test class checking NewOfferConsistencyCheck trigger
+**/
+@isTest
+private class NewOfferConsistencyCheckTest {
+	@isTest 
+  static void insertOfferVehiclePresent() {
+    
+    List<Showroom__c> showrooms = TestHelpers.createTestShowrooms(2);
+    Showroom__c showroom1 = showrooms[0];
+    Showroom__c showroom2 = showrooms[1];
+    Vehicle__c vehicle = TestHelpers.createTestVehicle();
+
+    Offer__c offer = new Offer__c(Vehicle__c = vehicle.Id, Showroom__c = showroom1.Id, Present__c = True);
+    insert offer;
+        
+    Test.startTest();
+      Offer__c testOffer = new Offer__c(Vehicle__c = vehicle.Id, Showroom__c = showroom2.Id, Present__c = True);
+      Database.SaveResult result = Database.insert(testOffer, false);
+    Test.stopTest();
+        
+    System.assert(!result.isSuccess());
+    System.assertEquals('Pojazd jest już dostępny w salonie ' + showroom1.Name, result.getErrors()[0].getMessage());
+  }
+
+  @isTest 
+  static void insertOfferVehicleNotPresent() {
```

Comment:
dodaj jdocki do klas i metod

Reply:
dodałem niepotrzebniej jakąś wtyczkę gitową i powrzucało historyczne pliki z folderu ".history". Naprawię to przez .gitignore